src/validation_workflow/validation.py

```diff
@@ -0,0 +1,24 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+
+from abc import ABC, abstractmethod
+
+
+class Validation(ABC):
+    base_url = "https://artifacts.opensearch.org/releases/bundle/"
+
+    def __init__(self, version: str, distribution: str, platform: str, projects: list) -> None:
+        self.version = version
+        self.distribution = distribution
+        self.platform = platform
+        self. projects = projects
```

Comment:
Typo, which would cause the code not running at all.

Reply:
This also means you are missing test cases because the test pass but this code is invalid.
Or it means there are variables you did not use.

Reply:
I haven't used the init method in base class for implementation.So tried removing the init as I don't need any class properties in particular.